frontend/public/components/utils/download-button.tsx

```diff
@@ -36,7 +37,7 @@ export const DownloadButton: React.FC<DownloadButtonProps> = (props) => {
       </span>
       <span style={{visibility: 'hidden'}}>&nbsp;{filename}</span>
     </button>
-    { error && <p className="alert alert-danger" style={{wordBreak: 'break-word'}}><span className="pficon pficon-error-circle-o"></span>{error.toString()}</p> }
+    { error && <Alert isInline className="co-alert" variant="danger" style={{wordBreak: 'break-word'}} title={error.toString()} /> }
```

Comment:
could we replace the inline `break-word` style with the `co-break-word` class https://github.com/openshift/console/blob/master/frontend/public/style/mixin/_break-word.scss